plugin/src/main/java/com/denizenscript/denizen/tags/core/ServerTagBase.java

```diff
@@ -2698,14 +2019,416 @@ public static void adjustServer(Mechanism mechanism) {
+        // @attribute <server.list_files[<path>]>
+        // @returns ListTag
+        // @deprecated use util.list_files
+        // @description
+        // Deprecated in favor of <@link tag util.list_files>
+        // -->
+
+        // <--[tag]
+        // @attribute <server.java_version>
+        // @returns ElementTag
+        // @deprecated use util.java_version
+        // @description
+        // Deprecated in favor of <@link tag util.java_version>
+        // -->
+
+        // <--[tag]
+        // @attribute <server.sql_connections>
+        // @returns ListTag
+        // @deprecated use util.sql_connections
+        // @description
+        // Deprecated in favor of <@link tag util.sql_connections>
+        // -->
+
+        // <--[tag]
+        // @attribute <server.scripts>
+        // @returns ListTag(ScriptTag)
+        // @deprecated use util.scripts
+        // @description
+        // Deprecated in favor of <@link tag util.scripts>
+        // -->
+
+        // <--[tag]
+        // @attribute <server.last_reload>
+        // @returns TimeTag
+        // @deprecated use util.last_reload
+        // @description
+        // Deprecated in favor of <@link tag util.last_reload>
+        // -->
+
+        // <--[tag]
+        // @attribute <server.stack_trace>
+        // @returns ElementTag
+        // @deprecated use util.stack_trace
+        // @description
+        // Deprecated in favor of <@link tag util.stack_trace>
+        // -->
+
+        // <--[tag]
+        // @attribute <server.debug_enabled>
+        // @returns ElementTag(Boolean)
+        // @deprecated use util.debug_enabled
+        // @description
+        // Deprecated in favor of <@link tag util.debug_enabled>
+        // -->
+        if (deprecatedServerUtilTags.contains(attribute.getAttributeWithoutParam(1))) {
+            BukkitImplDeprecations.serverUtilTags.warn(attribute.context);
+            return UtilTagBase.instance.getObjectAttribute(attribute);
+        }
+        return super.getObjectAttribute(attribute);
+    }
+
+    public static final Set<String> deprecatedServerUtilTags = Set.of("current_time_millis", "real_time_since_start",
+            "delta_time_since_start", "current_tick", "available_processors", "ram_usage", "ram_free", "ram_max", "ram_allocated", "disk_usage", "debug_enabled",
+            "disk_total", "disk_free", "started_time", "has_file", "list_files", "notes", "last_reload", "scripts", "sql_connections", "java_version", "stack_trace");
+
+    private static MapTag areaNotesDebugStreamHack(Int2ObjectOpenHashMap<NotedAreaTracker.AreaSet> set) {
+        MapTag out = new MapTag();
+        for (Int2ObjectMap.Entry<NotedAreaTracker.AreaSet> pair : set.int2ObjectEntrySet()) {
+            out.putObject(String.valueOf(pair.getIntKey()), new ListTag(pair.getValue().list.stream().map(t -> t.area).toList()));
+        }
+        return out;
+    }
+
+    public static void listDeprecateWarn(Attribute attribute) {

```

Comment:
This is redundant now I believe - it should already get an appropriate deprecation warning for the `list_` variants because they are being explicitly registered as deprecated variants